packages/gatsby-transformer-screenshot/lambda/__manual-tests__/local-test.js

```diff
@@ -0,0 +1,24 @@
+const thing = require(`../index`)
+const url = process.env.SITE_URL
+
+const start = async () => {
+  const result = await thing.handler(
+    {
+      body: `{"url":"${url}"}`,
+    },
+    {
+      fail: err => {
+        console.log(`error:`)
+        console.log(err)
+      },
+      succeed: res => {
+        console.log(`success!`)
+        console.log(res)
+      },
+    }
+  )
+
+  return result
+}
```

Comment:
small nits. you could omit all of the async/await since you are just returning the promise. It'll be the same thing if you just did `const start = () => thing.handler(...)`

Reply:
oh, and at that point since we just invoke it below you don't need any of this. just run `thing.handler` directly.

Also, what is `thing` ? lol

Reply:
totally fine to ignore these if you want to keep them stylistically aligned with other tests that require a async wrapper

Reply:
Should be okay for now, can be followed up if necessary

Reply:
> Also, what is thing ? lol

😅 it's the thing! that does the stuff! y'know? 

both fair points. This was (pretty scrappy) manual testing code that I nearly deleted. I left it in in case it's useful for the next person making changes to the Lambda function.